lib/action_policy/rspec/dsl.rb

```diff
@@ -77,7 +77,7 @@ def formatted_policy(policy)
         )
       end
     else
-      specify do
+      specify "is disallowed" do
```

Comment:
```suggestion
      specify "is not authorized" do
```

Reply:
The same comment regarding "authorized" as above.

My suggestion would be to use "is denied", since we already use this terminology (e.g., in pre-checks):

```suggestion
      specify "is denied" do
```